src/Parsers/Access/ASTExecuteAsQuery.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <Parsers/IAST.h>
+
+
+namespace DB
+{
+class ASTRolesOrUsersSet;
+
+/** EXECUTE AS <user>
+  */
```

Comment:
It would be nice to have a command to execute only one statement as a specified user, then get back to normal mode. Perhaps something like
```
EXECUTE AS <user> SELECT ...
```